app/views/catalog/_index.html.erb

```diff
@@ -63,7 +63,10 @@
       <%= render 'additional_results_block' %>
     <%- end %>
 
-    <%= render 'results_pagination' %>
+    <%- if !@response.empty? %>
```

Comment:
It might be more readable to use Ruby's `unless` instead of `if !{Boolean}`.

(also, and I know this is above, but just an FYI the `<%-` can be written simply as `<%`.  The `-` is a hold-over from previous versions of ERB and I try to remove whenever possible)